src/GovUk.Education.ExploreEducationStatistics.Admin/Models/Api/DataBlockViewModel.cs

```diff
@@ -21,7 +21,7 @@ public class DataBlockViewModel : IContentBlockViewModel
 
         public ObservationQueryContext DataBlockRequest { get; set; }
 
-        public List<IContentBlockChart> Charts { get; set; }
+        public IContentBlockChart Chart { get; set; }
```

Comment:
De-listing charts is actually a bad idea. A future feature is to be able to have multiple charts per data block. See: https://dfedigital.atlassian.net/browse/EES-753

I would revert all of the relevant changes.